resources/application.yaml

```diff
@@ -10,7 +10,7 @@ db:
   url: "sqlite:///database.db"
 
 storage:
-  studies: examples/studies
+  studies: /tmp/Antares_Simulator_Examples
```

Comment:
this file will be used to generate default config when launching application in local, le examples/studies be the default path for studies